ovirt_imageio/_internal/server.py

```diff
@@ -44,7 +45,7 @@ def main():
         log.info("Starting (hostname=%s pid=%s, version=%s)",
                  socket.gethostname(), os.getpid(), version.string)
 
-        server = Server(cfg)
+        server = Server(cfg, args.ticket)
```

Comment:
Using ticket=args.ticket is more clear and correct, since ticket is keyword argument. We always want to call with key=value style when using such arguments.